src/Symfony/Bridge/Twig/Tests/Extension/AbstractBootstrap3LayoutTest.php

```diff
@@ -2722,104 +2722,6 @@ public function testColor()
     [@name="name"]
     [@class="my&class form-control"]
     [@value="#0000ff"]
-'
-        );
-    }
-
-    public function testWeekSingleText()
```

Comment:
All those tests were copied from the parent class `AbstractLayoutTest`. They just check that the CSS class `form-control` is added. IMO, this is useless since the bootstrap themes do not override the `week_type` block. There is only one block in `form_div_layout.html.twig` that is tested in `AbstractLayoutTest`.

Reply:
Damn, just found out the override is needed because the xpath matching on the class is "strict": `@class="myclass"` does not match `class="myclass form-control"`.